homeassistant/components/rointe/climate.py

```diff
@@ -0,0 +1,275 @@
+"""Support for Rointe Climate."""
+
+from __future__ import annotations
+
+from rointesdk.device import RointeDevice
+
+from homeassistant.components.climate import (
+    PRESET_COMFORT,
+    PRESET_ECO,
+    ClimateEntity,
+    ClimateEntityDescription,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    CMD_SET_HVAC_MODE,
+    CMD_SET_PRESET,
+    CMD_SET_TEMP,
+    DOMAIN,
+    LOGGER,
+    PRESET_ROINTE_ICE,
+    RADIATOR_MODE_AUTO,
+    RADIATOR_MODE_MANUAL,
+    RADIATOR_PRESET_COMFORT,
+    RADIATOR_PRESET_ECO,
+    RADIATOR_PRESET_ICE,
+    RADIATOR_TEMP_MAX,
+    RADIATOR_TEMP_MIN,
+    RADIATOR_TEMP_STEP,
+)
+from .coordinator import RointeDataUpdateCoordinator
+from .rointe_entity import RointeRadiatorEntity
+
+AVAILABLE_HVAC_MODES: list[str] = [HVACMode.OFF, HVACMode.HEAT, HVACMode.AUTO]
+AVAILABLE_PRESETS: list[str] = [PRESET_COMFORT, PRESET_ECO, PRESET_ROINTE_ICE]
```

Comment:
Should these be enums instead of constants (to be consistent with `HVACMode`)? In general, we're moving away from "buckets" of constants to enums (so they are clearer, more explicit in typing, etc.).